examples/gno.land/r/faucet/faucet.gno

```diff
@@ -9,60 +9,88 @@ import (
 
 var (
 	// configurable by admin.
-	gAdminAddr       std.Address = "g1u7y667z64x2h7vc6fmpcprgey4ck233jaww9zq"
-	gControllerAddr  std.Address = "g1u7y667z64x2h7vc6fmpcprgey4ck233jaww9zq"
-	gPerTransferSend             = std.Coins{std.Coin{"ugnot", 1000000}}
-	gInPause                     = false
-	gMessage                     = "# Community Faucet.\n\n"
+	gAdminAddr   std.Address    = "g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5"
+	gControllers [3]std.Address = [3]std.Address{} // limit it to 3
```

Comment:
```suggestion
	gControllers []std.Address = []std.Address{}
```

why a limit of 3?

Reply:
1) I think we should always use an array instead of a slice to limit resource usage to avoid abuse. 

2) We can set the number to 100 if necessary. I think three faucet bot controllers are good enough for now, and we can replace each one of these controllers with provided API.

Reply:
I see your point, and it's a good idea to show good practices.

Let's update the limit to 10.

```suggestion
	gControllers [10]std.Address = [10]std.Address{} // limit it to 10
```